include/cutlass/epilogue/thread/linear_combination_relu.h

```diff
@@ -183,7 +183,27 @@ class LinearCombinationRelu {
       threshold_ = reinterpret_cast<ElementCompute const &>(allones);
     }
   }
-  
+
+  using ElementC = ElementOutput_;
+  using ElementD = ElementOutput_;
+
+  template<class TensorType>
+  CUTLASS_HOST_DEVICE
+  void operator()(TensorType &accumulators) const {
+    for (int i = 0; i < size(accumulators); i++) {
+      accumulators(i) = accumulators(i) < 0 ? 0 : accumulators(i);
+    }
+  }
+
+  template<class TensorDst, class TensorSrc>
+  CUTLASS_HOST_DEVICE
+  void operator()(TensorDst &accumulators,
+                  TensorSrc const &source) const {
+    for (int i = 0; i < size(accumulators); i++) {
+      accumulators(i) = accumulators(i) < 0 ? source(i) : accumulators(i) + source(i);
```

Comment:
Curious from a performance perspective, how does branching perform in contrast to say some sort of branchless code of this fashion (thought might not be a good suggestion)  - 
```
int le_zero = accumulators(i) < 0;
accumulators(i) = le_zero * accumulators(i) + (!le_zero) * (accumulators(i) + source(i));
```